docs/src/styles/docs/propsTable.scss

```diff
@@ -0,0 +1,17 @@
+.props-table-expander .amplify-expander__content__text {
+  overflow-x: auto;
+}
+
+.props-table {
+  & pre[class*='language-'] {
+    max-width: 20rem;
```

Comment:
Can we remove some of the extra spacing around pre in the table? Like:
```css
.props-table pre[class*=language-] {
   margin: 0;
   background: none;
   padding: 0;
}
   ```
Removes some of the unneeded space.

<img width="843" alt="Screen Shot 2022-06-13 at 10 28 12 AM" src="https://user-images.githubusercontent.com/376920/173376721-7d10a9dd-284d-41f4-bd65-e148241b3dd2.png">